README.rst

```diff
@@ -1,47 +1,30 @@
 Ray Tutorial
 ============
 
-Setup
------
+To run the tutuorial online, click on the binder link below:
 
-1. Make sure you have Python installed (we recommend using the `Anaconda Python
-   distribution`_). Ray works with both Python 2 and Python 3. If you are unsure
-   which to use, then use Python 3.
+.. image:: https://mybinder.org/badge.svg :target: https://mybinder.org/v2/gh/jzf2101/tutorial/master
```

Comment:
I'm seeing the following, so I'm not sure this is formatted correctly. 
<img width="808" alt="screen shot 2018-06-19 at 4 42 10 pm" src="https://user-images.githubusercontent.com/249517/41629699-cd746e3a-73df-11e8-9eff-f2b4d838e732.png">